src/main/java/pages/MoodUpdatedModalPage.java

```diff
@@ -1,4 +1,22 @@
 package pages;
 
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
 public class MoodUpdatedModalPage {
+
+    public static final String GO_TO_DIARY_BTN = ".btn-ssm";
+    public static final By UPDATED_MOOD_SUCCESS_TEXT = By.xpath("//*[@class='modal-body']/p");
+    //TODO: when i write public static final String UPDATED_MOOD_SUCCESS_TEXT = "//*[@class='modal-body']/p" the first "/" marked as unexpected token
```

Comment:
$() - by default accepts only CSS. If you'd like to pass xpath, you should specify $(By.xpath(.......))